workers/request-worker.js

```diff
@@ -111,7 +111,10 @@ function makeRequest (job, callback) {
         koop.Cache.insertPartial('agol', job.data.itemId, geojson, job.data.layerId, function (err) {
           if (err) {
             featureService.pageQueue.kill()
-            throw err
+            koop.log.error('err')
+            var error = new Error('Error inserting rows into the db')
+            error.type = 'db'
+            throw error
```

Comment:
Nope, that's intentional. I want them to be caught inside the domain error listener so they can trigger the callback that declares the job failed.